### PR TITLE
Streamlined theme and randomly generate colors

### PR DESCRIPTION
**Pull request by nickfarrow**
Instead of xmas https://github.com/chaincase-app/nolooking/pull/62,
Random color theme engaging, try it out

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -81,6 +80,21 @@ <h2>PayJoin here to open these channels</h2>
...
 		<script type="text/javascript">
+			function random_style() {
+				// Generate random colors for background, --primary and --secondary
+				// RGB triplet of (32 + $[0, 128]) for some base color, which is then reordered for the secondary color
+				let color_triplet = [32 + Math.floor(Math.random() * 128), 32 + Math.floor(Math.random() * 128), 32 + Math.floor(Math.random() * 128)];
+				let bg_image = `linear-gradient(
+	to bottom right,
+	rgb(${color_triplet.join(", ")}),
+	rgb(7, 10, 19)
+)`;
```
Comment (0xBEEFCAF3) [inline public/index.html:91]: Nit: indent to the previous line

**Review by 0xBEEFCAF3 — APPROVED** [review]
Nice!!!

File: public/index.html [hunk 2]
```diff
@@ -21,21 +20,21 @@
...
+			<a href="https://nolooking.chaincase.app/" target="_blank" style="text-decoration:none"><h1>⚡️&nbsp;<img src="banner.gif" style="display:inline">&nbsp;⚡️</h1></a>
+			<h2>Alpha [experimental] | Avoid sp👀ks</h2>
 		</header>
 		<main class="center-axyz">
 			<form action="/schedule" method="post" enctype="application/x-www-form-urlencoded"
-				x-flex direction="column">
-				<fieldset>
-					<legend>Queue Inbound Channel</legend>
+			x-flex direction="column">
+			<h3>Queue batches of lightning channels to open in a single transaction</h3>
```
[the remark this answers is not in the document]
Comment (DanGould) [reply public/index.html:29]: I think It's good if the form has a header that says what it's for ✅

File: public/style.css [hunk 3]
```diff
@@ -71,22 +60,33 @@ form > * {
...
-    margin: 2rem;
-}
+
 
 input {
     text-align: right;
 }
 
+legend {
+    color: var(--secondary);
```
Comment (DanGould) [inline public/style.css:70]: same comment as p, h1, h3 re: color: var(--color);

File: public/style.css [hunk 4]
```diff
@@ -71,22 +60,33 @@ form > * {
...
+
 .invisible {
     display: none;
 }
 
+.colored-button {
+    color: black;
+    background: var(--secondary);
+}
+
```
Comment (DanGould) [inline public/style.css:85]: You missed the checkbox by styling this with a custom class. When you inspect the buttons you applied this too, you can see they're already styled with `--buttonColor` and `--buttonBg`, which is the place for this change.

As-is this change is a regression

**Comment by DanGould** [comment]
This gets the job done

**Pull request merged**